atlasdb-workload-server/src/test/java/com/palantir/atlasdb/workload/transaction/WorkloadTestHelpers.java

```diff
@@ -34,6 +37,16 @@ public final class WorkloadTestHelpers {
     public static final TableReference INDEX_REFERENCE =
             TableReference.create(TABLE_REFERENCE.getNamespace(), TABLE_1_INDEX_1);
 
+    public static final Map<String, TableReference> NAMES_TO_REFERENCES_TABLE_1 = Map.of(
+            TABLE_1, TABLE_REFERENCE,
+            TABLE_1_INDEX_1, INDEX_REFERENCE);
+
+    public static final Map<TableReference, byte[]> TABLES_TO_ATLAS_METADATA = Map.of(
+            TABLE_REFERENCE,
+            AtlasDbUtils.tableMetadata(ConflictHandler.SERIALIZABLE),
+            INDEX_REFERENCE,
+            AtlasDbUtils.indexMetadata(ConflictHandler.SERIALIZABLE));
```

Comment:
Index tables should have `ConflictHandler.SERIALIZABLE_INDEX`, unless I'm missing something?

Reply:
This was a mistake!